app/apps/server/converters/roomSubscriptions.js

```diff
@@ -0,0 +1,41 @@
+import { Subscriptions } from '../../../models';
+
+export class AppRoomSubscriptionsConverter {
+	constructor(orch) {
+		this.orch = orch;
+	}
+
+	convertSubscriptionToAppById(subscriptionId) {
+		return this.convertSubscriptionToApp(Subscriptions.findOneById(subscriptionId));
+	}
+
+	convertSubscriptionToApp(subscription) {
+		if (!subscription) {
+			return undefined;
+		}
+
+		let user;
+		if (subscription.u && subscription.u._id) {
+			user = this.orch.getConverters().get('users').convertById(subscription.u._id);
+		}
+
+		let room;
+		if (subscription.rid) {
+			room = this.orch.getConverters().get('rooms').convertById(subscription.rid);
+		}
+
+		return {
```

Comment:
Please use the `transformMappedData` function, as seen in the [messages converter](https://github.com/RocketChat/Rocket.Chat/blob/develop/app/apps/server/converters/messages.js#L75)